py/server/deephaven/ugp.py

```diff
@@ -81,14 +83,36 @@ def do_locked(*arg, **kwargs):
     return do_locked
 
 
+def _check_arg(arg):
+    if isinstance(arg, abc.Sequence):
+        for e in arg:
+            if _check_arg(e):
+                return True
+    elif getattr(arg, "is_refreshing", False):
+        return True
+
+    return False
+
+
+def _has_refreshing_tables(*args, **kwargs):
+    for arg in args:
+        if _check_arg(arg):
+            return True
+    for k, v in kwargs.items():
+        if _check_arg(v):
+            return True
+
+    return False
+
+
```

Comment:
Maybe a better name is warranted?
```suggestion
def _is_arg_refreshing(arg):
    if isinstance(arg, abc.Sequence):
        for e in arg:
            if _is_arg_refreshing(e):
                return True
    elif getattr(arg, "is_refreshing", False):
        return True

    return False


def _has_refreshing_tables(*args, **kwargs):
    for arg in args:
        if _is_arg_refreshing(arg):
            return True
    for k, v in kwargs.items():
        if _is_arg_refreshing(v):
            return True

    return False


```